MembershipApp/templates/MembershipApp/navbars/anonymous_navbar.html

```diff
@@ -0,0 +1,97 @@
+{% load static %}
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="{% static "MembershipApp/css/stack-interface.css" %}" rel="stylesheet" type="text/css" media="all">
+    <link href="{% static "MembershipApp/css/bootstrap.css" %}" rel="stylesheet" type="text/css" media="all"/>
+    <link href="{% static "MembershipApp/css/stack-interface.css" %}" rel="stylesheet" type="text/css" media="all"/>
+    <link href="{% static "MembershipApp/css/theme.css" %}" rel="stylesheet" type="text/css" media="all"/>
+    <link href="{% static "MembershipApp/css/custom.css" %}" rel="stylesheet" type="text/css" media="all"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:200,300,400,400i,500,600,700" rel="stylesheet">
+
```

Comment:
this is weird, if you want to have to different base.html one for the static and one for the app(after login) please reformat the file. 
Navbar should only have html for the navbar